javascript/acts.js

```diff
@@ -3,21 +3,21 @@ const actsContent = [
     card: 1,
     image: 'images/photo1.jpeg',
     icon: 'images/music.png',
-    title: 'Black Tea',
+    title: 'Black Coffee',
     subHeading: 'DJ, Record Producer & Songwriter',
-    description: `Black Tea is a South African DJ, record producer, and
+    description: `Black Coffee is a South African DJ, record producer, and
                 songwriter. He's one of the most influential acts on the
                 continent known exclusively for his soulful Afro-House sound.
-                Tea has established his own record label, Soulistic Music, and
-                has recently released a self-titled album called Black Tea.`,
+                Coffee has established his own record label, Soulistic Music, and
+                has recently released a self-titled album called Black Coffee.`,
   },
   {
     card: 2,
     image: 'images/photo2.jpg',
     icon: 'images/music.png',
-    title: 'Wiz',
+    title: 'WizKid',
     subHeading: 'Singer & Songwriter',
-    description: 'Wiz is a Nigerian singer and songwriter. He has multiple awards including a grammy and is considered one of the greatest African artists. By February 2021, Wiz was the most-streamed Nigerian artist of all time on Spotify, with over 3.4 billion streams across all credits.',
+    description: 'WizKid is a Nigerian singer and songwriter. He has multiple awards including a grammy and is considered one of the greatest African artists. By February 2021, Wiz was the most-streamed Nigerian artist of all time on Spotify, with over 3.4 billion streams across all credits.',
   },
   {
     card: 3,
```

Comment:
The section "Featured speakers" is generated dynamically with JavaScript 🔥 Superb 🚀

Reply:
You created a function that iterates through the act's list and creates HTML elements with the data from each object 🤯